techniques/systemSettings/misc/clockConfiguration/4.0/clockConfiguration.st

```diff
@@ -0,0 +1,462 @@
+
+    pass3.not_repaired_susetz::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_error", "&TRACKINGKEY&", "Time zone", "None", "Could not update SuSE-style time zone parameters (/etc/sysconfig/clock)");
+
+    pass3.susetz_kept::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "log_info", "&TRACKINGKEY&", "Time zone", "None", "SuSE-style time zone parameters already correct (/etc/sysconfig/clock)");
+
+    # Global reports for the "Hardware clock (RTC)" component
+    pass3.(!windows.!ntp_hwclock_sync_error.!ntp_hwclock_synced.clock_hwclock_sync.clock_hwclock_has_rtc)::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_success", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "It is not yet time to synchronize the hardware clock with the NTP time. Skipping...");
+
+    pass3.ntp_hwclock_synced::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_success", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "The hardware clock has been synchronized with the NTP time");
+
+    pass3.ntp_hwclock_sync_error::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_error", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "The hardware clock could not be synchronized with the NTP time");
+
+    pass3.clock_hwclock_sync.!clock_hwclock_has_rtc::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_na", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "No hardware clock present on /dev/rtc");
+
+    pass3.windows.!clock_hwclock_rtc2utc::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_success", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "The hardware clock is automatically synchronized with the NTP time on Windows");
+
+    pass3.rtc_regkept::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_success", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "RTC set to UTC");
+
+    pass3.rtc_regfail::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_error", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "RTC failed to be set to UTC");
+
+    pass3.rtc_regkept::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "log_info", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "RTC already set to UTC");
+
+    pass3.!clock_hwclock_sync::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_success", "&TRACKINGKEY&", "Hardware clock (RTC)", "None", "No synchronization with the hardware clock was requested");
+
+    pass3.!clock_date_s_now::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_na", "&TRACKINGKEY&", "Xen Time", "None", "Setting Time is disabled");
+
+    pass3.clock_date_s_now.!xen_dom0::
+      "any" usebundle => rudder_common_report("ntpConfiguration", "result_na", "&TRACKINGKEY&", "Xen Time", "None", "Not on a Xen Dom0");
+
+    pass3.date_s_set::
```

Comment:
I am throwing out this "date -s now" code as it's too specific for our problem, and is not ever 100% correct in some corner-cases, and would only confuse people, so this will be gone in the next version of the PR.